mmv1/products/vertexai/FeatureOnlineStore.yaml

```diff
@@ -41,65 +43,183 @@ description: Vertex AI Feature Online Store provides a centralized repository fo
 autogen_async: false
 examples:
   - !ruby/object:Provider::Terraform::Examples
-    name: vertex_ai_feature_online_store
-    primary_resource_id: feature_online_store
+    name: 'vertex_ai_featureonlinestore_with_bigtable'
+    primary_resource_id: 'featureonlinestore'
+    primary_resource_name: "fmt.Sprintf(\"terraform%s\",
+      context[\"random_suffix\"\
+      ])"
     vars:
-      name: example_feature_online_store
+      name: 'terraform2'
```

Comment:
These variables need to have a `-` or `_` in order for them to behave properly in tests (it signals that a `tf-test` prefix and a random suffix should be added). I would suggest sticking with the prior name here because it is more descriptive, but you could change to something else as well. Same for other variables that are identifiers.